app/src/test/java/gov/va/vro/controller/MasControllerTest.java

```diff
@@ -152,22 +149,10 @@ void automatedClaimInScope() throws Exception {
 
     var request = MasTestData.getMasAutomatedClaimRequest(567, "7101", "999");
     var responseEntity = post("/v2/automatedClaim", request, MasResponse.class);
-    assertEquals(HttpStatus.OK, responseEntity.getStatusCode());
+    assertEquals(HttpStatus.UNPROCESSABLE_ENTITY, responseEntity.getStatusCode());
     mockMasOfframpEndpoint.expectedMessageCount(1);
 
     verifyClaimPersisted(request);
```

Comment:
I noticed the test above you return and dont verify persisted (assuming we dont persist?) but here we check. Wouldn't both 422 codes not persist (or both persist?)

Reply:
this may be my misunderstanding of specification if that's not true. if not, maybe a comment?